python/src/test/java/org/apache/zeppelin/python/PythonInterpreterTest.java

```diff
@@ -105,4 +106,28 @@ public void testCancelIntp() throws InterruptedException, InterpreterException {
     t.join(2000);
     assertFalse(t.isAlive());
   }
+
```

Comment:
Could you move them to BasePythonInterpreterTest#testPythonBasics ?

Reply:
Done